robottelo/ui/activationkey.py

```diff
@@ -40,13 +39,10 @@ def create(self, name, env, limit=None, description=None,
                     .format(name)
                 )
             if content_view:
-                Select(
-                    self.find_element(locators['ak.content_view'])
-                ).select_by_visible_text(content_view)
+                self.select(locators['ak.content_view'], content_view)
             else:
-                Select(
-                    self.find_element(locators['ak.content_view'])
-                ).select_by_value('0')
+                self.select(
+                    locators['ak.content_view'], 'Default Organization View')
```

Comment:
It doesn't seem so critical to use direct cv name here and if ever it become an issue in future - we can always expand our framework to support not only select lists text